src/snowflake/cli/_plugins/nativeapp/codegen/compiler.py

```diff
@@ -37,10 +41,12 @@
 
 SNOWPARK_PROCESSOR = "snowpark"
 NA_SETUP_PROCESSOR = "native app setup"
+TEMPLATING_PROCESSOR = "templating"
```

Comment:
a little bikeshedding necessary for 'templating' here. How do we refer to the templates feature in public docs again?

Reply:
or `template expansion` as another option

Reply:
updated

Reply:
trivial but can you update `templating` to `templates` in the pr description as well :')